lib/analytics_events_documenter.rb

```diff
@@ -129,6 +129,11 @@ def missing_documentation
         errors << "#{error_prefix} #{tag.name} missing types" if !tag.types
       end
 
+      description = method_description(method_object)
+      if description.present? && !method_object.docstring.match?(/\A[A-Z]/)
+        errors << "#{error_prefix} method description starts with lowercase, check indentation"
+      end
```

Comment:
Could we include the parsed description in the error message so it'll be even more obvious if the description includes param text? For context, I was previously unaware of the indentation requirement, so "check indentation" probably wouldn't have been enough for me to realize if a description includes parameter text.

Reply:
How about 24e3453036c8554f5721723e5f41e1d286ff9ab7, which would look like this:

```
app/services/analytics_events.rb:249 add_email_confirmation method description starts with lowercase, check indentation:
  selection.
  A user has clicked the confirmation link in an email"
```

Reply:
wfm 👍